script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -240,12 +239,12 @@ W-What? How the heck was she able to stop Arcueid with a half-hearted charge lik
 [sha:6b6570277053785c73ff635655d62d1779d7afec]{
 -- Page 641, Offset 25981. Mods: @k, @e.
 -- 「おう、やればできるじゃねえか。給料分しっかり働けよ。
-"Oh, so you %{i}can%{/i} do it if you try. Go work hard and earn your pay.
+"Ey, see? You %{i}can%{/i} do it if you try. C'mon then, go earn your pay.
```

Comment:
Ey, I'm walkin here

Reply:
For the record, Italian-American is literally what I intended him to sound like.